src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,11 @@ public static void printLongestWordTop100() throws IOException {
 				.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
 		List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 		
-		// TODO 이 부분에 구현한다.
+		words.stream().filter(s -> s.length() > 12)
+				      .sorted((o1, o2) -> o2.length() - o1.length())
```

Comment:
Comparator API를 활용하여 정렬하는 것은 어떨까요

```suggestion
				      .sorted(comparing(String::length))
```